src/lib/uri/uri.h

```diff
@@ -13,6 +13,52 @@
 extern "C" {
 #endif /* defined(__cplusplus) */
 
+struct uri_option_value {
+        /** Each URI option value can be represented by string. */
+        char *value;
+};
```

Comment:
What's the point to have this as a separate struct? Why not inline it in `uri_option_values`?